.cargo/config.toml

```diff
@@ -31,3 +31,14 @@ rustflags = [
   "-Aclippy::stable_sort_primitive",     # prefer stable sort
   "-Aclippy::extra-unused-type-parameters", # stylistic
 ]
+
+[env]
+# The version of the node.
+#
+# This is the version that is used for versioning this node binary.
+# By default the `minor` version is bumped in every release. `Major` or `patch` releases are only
+# expected in very rare cases.
+#
+# The worker binaries associated to the node binary should ensure that they are using the same
+# version as the main node that started them.
+POLKADOT_NODE_VERSION = "1.1.0"
```

Comment:
@bkchr Moved the node version here to avoid depending on `polkadot-cli`. Needed because in PVF test code I added a version check, see `get_and_check_worker_paths`. Maybe the version could be in a helper crate instead, wasn't sure where to put it.

Reply:
I would rather opt in for a helper crate. While this also don't being the best idea, it is probably better than the env variable.

But why not just depend on `polkadot-cli` as dev dependency? What is the problem?

Reply:
> But why not just depend on polkadot-cli as dev dependency? What is the problem?

@bkchr It can't be a dev-dependency. Since the new utility is exported to integration tests and benches, we can't gate it with `#[cfg(test)]` or it won't get exported. So it's behind a `test-utils` feature flag, which uses the production dependencies (though it should never end up in production).

I just tried making the dep optional and depending on it in the `test-utils` feature, but it was somehow introducing a cyclic dependency. I moved some things around but couldn't break the cycle easily. Probably due to `cargo`'s feature unification of one of the deps somewhere.